pages/account/manage/profile.js

```diff
@@ -218,7 +221,17 @@ export default function Profile({ BASE_URL, profile, fileExists }) {
                         />
                       </div>
                     </div>
-
+                    <div className="col-span-3 sm:col-span-4">
+                      <div className="mt-1">
+                        <Select
+                            name="pronoun"
+                            label="Pronouns"
+                            value={pronoun}
+                            options={pronouns}
+                            onChange={(e) => setPronoun(e.target.value)}
+                        />
```

Comment:
The indentation doesn't match project style, please check Prettier plugin

```suggestion
                        <Select
                          name="pronoun"
                          label="Pronouns"
                          value={pronoun}
                          options={pronouns}
                          onChange={(e) => setPronoun(e.target.value)}
                        />
```